client/Friend.js

```diff
@@ -68,4 +68,9 @@ Friend.prototype.update = function(info) {
 		if(info.position.coords.accuracy)
 			this.accuracyCircle.setRadius(info.position.coords.accuracy)
 	}
+	this.updateMarker(info.stale)
+}
+Friend.prototype.updateMarker = function(stale) {
+	var icon = stale ? 'http://maps.google.com/mapfiles/ms/icons/green.png' : ''
```

Comment:
I've got a branch that generates icons on the client, so this is easily resolved.